### ISSUE TITLE
logs spammed by warnings during initial sync

### ISSUE THREAD

**Issue by AskAlexSharov** [ledgerwatch/erigon#10203]
@battlmonstr hi. after https://github.com/ledgerwatch/erigon/pull/10180 - during genesis sync i see non-stop warnings: 
```
[WARN] [05-04|11:54:39.650] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:39.750] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:39.850] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:39.950] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:40.051] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:40.151] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:40.251] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:40.351] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[INFO] [05-04|11:54:40.443] [applying step 1404][agg] computing trie progress=1074k/1859k alloc=17.7GB sys=24.0GB
[WARN] [05-04|11:54:40.451] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:40.551] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:40.652] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:40.752] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:40.853] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:40.953] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.054] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.155] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.255] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.322] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.422] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.522] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.623] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.723] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.823] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:41.925] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.025] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.125] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.225] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.325] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.426] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.526] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.626] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.726] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.827] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:42.928] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.028] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.128] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.228] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.329] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.429] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.530] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.631] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.732] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.832] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:43.933] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.033] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.134] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.235] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.335] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.436] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.536] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.637] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.738] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.838] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:44.938] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.038] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.139] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.239] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.339] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.439] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.539] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.640] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.740] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.840] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:45.940] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.040] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.141] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.241] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.341] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.441] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.542] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.642] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.742] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.842] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:46.943] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.043] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.143] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.243] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.344] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.444] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.545] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.645] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.745] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.846] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:47.946] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.047] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.147] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.247] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.348] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.448] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.548] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.648] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.748] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.849] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:48.949] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.049] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.150] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.250] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.350] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.450] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.551] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.651] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.752] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.852] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:49.952] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.053] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.153] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.253] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.353] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.454] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.555] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.656] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.756] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.856] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:50.957] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.058] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.159] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.260] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.324] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.425] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.525] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.625] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.726] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.826] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:51.927] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.027] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.128] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.228] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.328] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.429] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.530] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.630] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.730] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.830] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:52.931] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.032] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.132] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.233] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.333] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.433] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.534] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.634] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.734] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.834] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:53.935] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.035] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.135] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.236] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.336] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.436] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.536] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.637] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.738] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.838] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:54.939] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.039] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.140] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.240] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.340] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.440] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.541] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.641] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.741] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.842] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:55.942] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.042] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.143] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.243] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.343] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.444] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.545] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.645] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.745] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.846] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:56.946] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.047] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.148] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.248] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.349] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.449] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.550] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.650] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.750] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.851] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:57.951] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.052] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.152] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.253] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.353] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.453] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.554] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.655] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.755] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.855] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:58.956] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.057] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.158] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.258] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.359] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.460] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.561] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.661] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.762] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.862] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:54:59.962] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:55:00.063] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:55:00.163] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:55:00.263] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:55:00.363] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[INFO] [05-04|11:55:00.442] [applying step 1404][agg] computing trie progress=1147k/1859k alloc=22.3GB sys=24.0GB
[WARN] [05-04|11:55:00.463] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:55:00.564] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:55:00.664] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
[WARN] [05-04|11:55:00.764] ethereumExecutionModule.InsertBlocks: ExecutionStatus_Busy
```

**Comment by AskAlexSharov** [comment]
log line 
```
[INFO] [05-04|11:54:40.443] [applying step 1404][agg] computing trie progress=1074k/1859k alloc=17.7GB sys=24.0GB
```
means Execution stage is in progress

**Comment by AskAlexSharov** [comment]
If `grep -v`, I still see other warnings:  
```
[WARN] [05-04|11:48:34.923] ethereumExecutionModule.Ready: ExecutionStatus_Busy
[INFO] [05-04|11:48:37.277] [4/12 Execution] Transaction replay      blk=18577642 tx/s=2550.8 buffer=188.4MB/1.0GB stepsInDB=1.00 step=1404.8 alloc=14.8GB sys=21.8GB
[INFO] [05-04|11:48:57.292] [4/12 Execution] Transaction replay      blk=18578056 tx/s=3239.7 buffer=196.9MB/1.0GB stepsInDB=1.00 step=1404.8 alloc=15.0GB sys=22.7GB
[INFO] [05-04|11:49:02.574] P2P                                      app=caplin peers=63
[INFO] [05-04|11:49:03.582] [snapshots] state merge done accounts(val:1400-1404, hist:1400-1404, idx:1400-1404), storage(val:1400-1404, hist:1400-1404, idx:1400-1404), code(val:1400-1404, hist:1400-1404, idx:1400-1404), commitment(val:1400-1404), logAddr=1400-1404, logTopic=1400-1404, traceFrom=1400-1404, traceTo=1400-1404
[WARN] [05-04|11:49:04.922] ethereumExecutionModule.Ready: ExecutionStatus_Busy
[INFO] [05-04|11:49:17.250] [4/12 Execution] Transaction replay      blk=18578498 tx/s=3512.0 buffer=205.6MB/1.0GB stepsInDB=1.00 step=1404.9 alloc=13.0GB sys=22.7GB
[WARN] [05-04|11:49:34.922] ethereumExecutionModule.Ready: ExecutionStatus_Busy
[INFO] [05-04|11:49:37.260] [4/12 Execution] Transaction replay      blk=18578969 tx/s=3721.9 buffer=215.0MB/1.0GB stepsInDB=1.00 step=1404.9 alloc=11.2GB sys=22.7GB
[INFO] [05-04|11:49:57.276] [4/12 Execution] Transaction replay      blk=18579451 tx/s=3842.3 buffer=224.2MB/1.0GB stepsInDB=1.00 step=1405.0 alloc=18.5GB sys=22.7GB
[INFO] [05-04|11:50:02.574] P2P                                      app=caplin peers=71
[WARN] [05-04|11:50:04.923] ethereumExecutionModule.Ready: ExecutionStatus_Busy
[INFO] [05-04|11:50:29.713] [Caplin-Blocks] Flushed buffer file      name=erigon-sortable-buf-1026288267
[WARN] [05-04|11:50:34.922] ethereumExecutionModule.Ready: ExecutionStatus_Busy
[INFO] [05-04|11:50:40.442] [applying step 1404][agg] computing trie progress=81k/1859k alloc=12.2GB sys=22.7GB
[INFO] [05-04|11:51:00.443] [applying step 1404][agg] computing trie progress=165k/1859k alloc=16.7GB sys=22.7GB
```

**Comment by yperbasis** [comment]
Fixed by PR #10211
